app/views/user_mailer/admin_deleted_work_notification.html.erb

```diff
@@ -1,11 +1,11 @@
 <% content_for :message do %>
-  <p>Dear <%= @user.default_pseud.byline %>,</p>
+  <p><%= t '.hello', user: @user.default_pseud.byline %></p>
   
-  <p>Your work <i><%= style_bold(@work.title.html_safe) %></i> was deleted from the Archive by a site admin.</p>
+  <p><%= t('.html.part1', title: style_bold(@work.title.html_safe)).html_safe %></p>
```

Comment:
I am not sure that is a better idea ( if you mean putting the markup in the .html.part1 bit.